pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/UfoerOmregningEnslig.kt

```diff
@@ -0,0 +1,1070 @@
+    paragraph {
+        textExpr(
+            Bokmal to "Du er sikret minsteytelse fordi beregningen ut fra din egenopptjente inntekt er lavere enn minstenivået for uføretrygd. Satsen på minsteytelsen avhenger av sivilstand. For deg utgjør minsteytelsen ".expr() + minsteytelse_sats_vedvirk.format() + " ganger folketrygdens grunnbeløp. Du kan lese mer om grunnbeløp på nav.no.".expr(),
+            Nynorsk to "Du er sikra minsteyting fordi utrekninga ut frå den eigenopptente inntekta di er lågare enn minstenivået for uføretrygd. Satsen på minsteytinga avheng av sivilstand. For deg utgjer minsteytinga ".expr() + minsteytelse_sats_vedvirk.format() + " gonger grunnbeløpet i folketrygda. Du kan lese meir om grunnbeløp på nav.no.".expr(),
+            English to "You are eligible for the minimum benefit, because the calculated benefit based on your income is lower than the minimum benefit. The rate of the minimum benefit depends on your marital status. Your minimum benefit is ".expr() + minsteytelse_sats_vedvirk.format() + " times the National Insurance basic amount. You can read more about the National Insurance basic amount at nav.no.".expr()
+        )
+    }
+    paragraph {
+        textExpr(
+            Bokmal to "Dette kan ha betydning for kompensasjonsgraden din som er satt til ".expr() + kompensasjonsgrad_ufoeretrygd_vedvirk.format() + " prosent. Du kan lese mer om dette i vedlegget.".expr(),
+            Nynorsk to "Dette kan ha noko å seie for kompensasjonsgraden din som er fastsett til ".expr() + kompensasjonsgrad_ufoeretrygd_vedvirk.format() + " prosent. Du kan lese meir om dette i vedlegget.".expr(),
+            English to "This may affect your degree of compensation, which is determined to be ".expr() + kompensasjonsgrad_ufoeretrygd_vedvirk.format() + " percent. You can read more about this in the appendix.".expr()
+        )
+    }
+}
+
+
+data class EndrMYOgMinstIFUDodEPS2_001Dto(
+    val minsteytelse_sats_vedvirk: Double,
+    val inntekt_foer_ufoerhet_vedvirk: Kroner,
+    val oppjustert_inntekt_foer_ufoerhet_vedvirk: Kroner,
+    val kompensasjonsgrad_ufoeretrygd_vedvirk: Double
+)
+
+val EndrMYOgMinstIFUDodEPS2_001 = OutlinePhrase<LangBokmalNynorskEnglish, EndrMYOgMinstIFUDodEPS2_001Dto> { dto ->
+    val minsteytelse_sats_vedvirk = dto.select(EndrMYOgMinstIFUDodEPS2_001Dto::minsteytelse_sats_vedvirk)
+
+
+    val kompensasjonsgrad_ufoeretrygd_vedvirk =
+        dto.select(EndrMYOgMinstIFUDodEPS2_001Dto::kompensasjonsgrad_ufoeretrygd_vedvirk)
+    paragraph {
+        textExpr(
+            Bokmal to "Du er sikret minsteytelse fordi beregningen ut fra din egenopptjente inntekt er lavere enn minstenivået for uføretrygd. Satsen på minsteytelsen avhenger av sivilstand. For deg utgjør minsteytelsen ".expr() + minsteytelse_sats_vedvirk.format() + " ganger folketrygdens grunnbeløp. Du kan lese mer om grunnbeløp på nav.no.".expr(),
```

Comment:
Jeg er for brede linjer, men dette er kanskje litt i det lengste laget? 😅

Reply:
tok frem hekkesaksa og fiksa litt 😄